src/Routes/RosPage/RosPage.js

```diff
@@ -79,6 +80,14 @@ class RosPage extends React.Component {
                 stateFilterValue: ['Undersized', 'Oversized', 'Under pressure', 'Idling']
             });
         }
+
+        this.setState({
+            osFilterValue: [
+                'RHEL 7.0', 'RHEL 7.1', 'RHEL 7.2', 'RHEL 7.3', 'RHEL 7.4', 'RHEL 7.5', 'RHEL 7.6',
+                'RHEL 7.7', 'RHEL 7.8', 'RHEL 7.9', 'RHEL 8.0', 'RHEL 8.1', 'RHEL 8.2', 'RHEL 8.3',
+                'RHEL 8.4', 'RHEL 8.5'
+            ]
+        });
```

Comment:
@apuntamb can you please remove this.setState from here as we discussed `processQueryParams` method hands the query params which are part of browser URL and we only need to handle it for state when the user routes from Dashboard links to ROS page.